README.md

```diff
@@ -1241,6 +1241,7 @@ auth_opt_mongo_password pwd
 auth_opt_mongo_users users_collection_name
 auth_opt_mongo_acls acls_collection_name
 auth_opt_mongo_disable_superuser true
+auth_opt_mongo_use_tls true
```

Comment:
```suggestion
auth_opt_mongo_with_tls true
```
That would be consistent with how we call the option in `http` and `jwt` backends.